### PR TITLE
#156882153 Grunt to Gulp

### PR DESCRIPTION
**Pull request by kenoseni**
#### What does this PR do?
Converts the grunt file to gulp file

#### Description of Task to be completed?
The gulp file created should be able to start the app

#### How should this be manually tested?
Clone the repo and cd into it. Run *npm run start:dev*

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
#156882153

#### Screenshots (if appropriate)
None

#### Questions:
None

### CODE REVIEW

File: app/controllers/avatars.js [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
-exports.allJSON = function(req, res) {
+exports.allJSON = function (req, res) {
   // Only return the first 12
   // The last 4 are reserved for guests
-  res.jsonp(avatars.slice(0,12));
+  res.jsonp(avatars.slice(0, 12));
 };
 
-exports.all = function() {
+exports.all = function () {
```
[the remark this answers is not in the document]
Comment (kenoseni) [reply app/controllers/avatars.js]: Please note that this error has now been fixed

File: app/controllers/avatars.js [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
+  '/img/chosen/H01.png',
+  '/img/chosen/J01.png',
+  '/img/chosen/M05.png',
+  '/img/chosen/N02.png',
+  '/img/chosen/N03.png',
+  '/img/chosen/N04.png',
+  '/img/chosen/N05.png'];
 
-exports.allJSON = function(req, res) {
+exports.allJSON = function (req, res) {
```
[the remark this answers is not in the document]
Comment (kenoseni) [reply app/controllers/avatars.js]: Please note that this error has now been fixed.

**Review by Spectrumsun — COMMENTED** [review]
Nice. NO Merge Conflict and hound error

File: .babelrc [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+    "presets" : [
+        "es2015"    
+    ]
+}
```
Comment (emmabaye) [inline .babelrc]: Nice work. However, there should be a new line at the end of this file.  Always look out for details like this.

File: app/controllers/avatars.js [hunk 4]
```diff
@@ -1,29 +1,27 @@
...
+exports.allJSON = (req, res) => {
   // Only return the first 12
   // The last 4 are reserved for guests
-  res.jsonp(avatars.slice(0,12));
+  res.jsonp(avatars.slice(0, 12));
 };
 
-exports.all = function() {
-  return avatars;
-};
```
Comment (emmabaye) [inline app/controllers/avatars.js:29]: Good job. However, there should be a new line at the end of this file.

File: .babelrc (hunk 3 above)
Comment (kenoseni) [reply .babelrc]: Thanks for the feedback, I will fix the newline at the end of this file.

File: app/controllers/avatars.js (hunk 4 above)
Comment (kenoseni) [reply app/controllers/avatars.js:29]: Thanks for this feedback, however, I have fixed this issue. Please refer to the latest pull request.

**Pull request merged**